src/grid/ref/grid_ref_collint.h

```diff
@@ -396,13 +396,13 @@ static inline void general_ci_to_reg(const int lp, const double exp_aiibic,
  * \brief Collocates coefficients C_i onto the grid for general case.
  * \author Ole Schuett
  ******************************************************************************/
-static inline void
-general_ci_to_grid(const int lp, const int j, const int jg, const int k,
-                   const int kg, const int npts_local[3], const int bounds_i[2],
-                   const int index_min[3], const int map_i[], const double zetp,
-                   const double dh[3][3], const double gp[3],
-                   const double radius, GRID_CONST_WHEN_COLLOCATE double *ci,
-                   GRID_CONST_WHEN_INTEGRATE double *grid) {
+static inline void general_ci_to_grid(
+    const int lp, const int jg, const int kg, const int ismin, const int ismax,
+    const int npts_local[3], const int bounds_i[2], const int index_min[3],
+    const int map_i[], const double zetp, const double a, const double b,
+    const double c, const double exp_ab, const double exp_2a,
+    const double gp[3], GRID_CONST_WHEN_COLLOCATE double *ci,
+    GRID_CONST_WHEN_INTEGRATE double *grid) {
 
   //--------------------------------------------------------------------
```

Comment:
Maybe this comment should be moved alongside.

Reply:
Yes, that was my concern too. I will put attach it to the call-side where this work now takes place.